packages/react/src/themes/teams/components/Popup/popupContentStyles.ts

```diff
@@ -9,6 +9,7 @@ const popupContentStyles: ComponentSlotStylesPrepared<PopupContentProps, PopupCo
     borderRadius: v.borderRadius,
     boxShadow: v.boxShadow,
 
+    width: 'fit-content',
```

Comment:
It seems that it will not work for IE:

![S91030-15034276](https://user-images.githubusercontent.com/14183168/67864867-a1006f00-fb26-11e9-920c-562f6e9e5092.jpg)
https://developer.mozilla.org/en-US/docs/Web/CSS/width

Reply:
But this is just a workaround for #2067 which is a workaround for bug in Chrome 69.
@jurokapsiar - what if we do #2067 conditionally only in Chrome?

Reply:
agree - in chrome and presumably in all chromium browsers (new edge)

Reply:
Both @levithomason and @dzearing are strongly against browser-specific code